packages/alert/src/index.vue

```diff
@@ -63,13 +77,13 @@ export default defineComponent({
     showIcon: Boolean,
     center: Boolean,
     effect: {
-      type: String,
+      type: String as PropType<AlertEffect>,
       default: 'light',
       validator: (value: string): boolean => ['light', 'dark'].indexOf(value) > -1,
     },
   },
   emits: ['click'],
-  setup(props, ctx) {
+  setup(props: IAlertProps, ctx) {
```

Comment:
useless interface for setup Props

Reply:
Typescript automatically infers props' type for user if you have adapted `defineComponent`